ethereum/gnosis_safe/view_safes.sql

```diff
@@ -22,6 +22,8 @@ CREATE MATERIALIZED VIEW gnosis_safe.view_safes AS
     SELECT contract_address AS address, evt_block_time AS creation_time
     FROM gnosis_safe."GnosisSafev1.3.0_evt_SafeSetup";
 
+CREATE UNIQUE INDEX CONCURRENTLY IF NOT EXISTS view_safes_unique_idx ON gnosis_safe.view_safes (address);
```

Comment:
👍🏻

Reply:
That was an edge case, I wasn't aware of. Should have checked it beforehand, sorry for that.

Reply:
Should be fixed now. I checked uniqueness with this query: https://dune.xyz/queries/538701